starter-code/scripts/articleView.js

```diff
@@ -85,25 +86,37 @@ articleView.initNewArticlePage = function() {
   });
 
   // TODO: Add an event handler to update the preview and the export field if any inputs change.
-
+  // listenToWhat, listenForWhat, doWhat
+  $('#form').on('change', articleView.create);
 };
 
 articleView.create = function() {
   // TODO: Set up a var to hold the new article we are creating.
   // Clear out the #articles element, so we can put in the updated preview
-
+  var article;
+  $('#articles').empty();
 
   // TODO: Instantiate an article based on what's in the form fields:
+  article = new Article({
+    author: $('#article-author').val(),
+    authorUrl: $('#article-author-url').val(),
+    title: $('#article-title').val(),
```

Comment:
When you made the form you specified the Id as 'title' and here you have it as #article-title which is causing the title not to show up in your preview or in your json object.